shared/chat/conversation/messages/react-button/index.js

```diff
@@ -15,6 +16,54 @@ import {Picker} from 'emoji-mart'
 import {backgroundImageFn} from '../../../../common-adapters/emoji'
 import DelayInterval from './delay-interval'
 
+type WrapProps = {|
+  backgroundImageFn: (set: string, sheetSize: string) => string,
+  onClick: ({colons: string}, evt: Event) => void,
+|}
+
+class PickerWrapper extends React.Component<WrapProps> {
+  _picker: Picker
+
+  // Setting autoFocus={true} on Picker doesn't work, so focus it
+  // ourselves on mount/update.
+
+  componentDidMount = () => {
+    this._focus()
+  }
+
+  componentDidUpdate = () => {
+    this._focus()
+  }
+
+  _focus = () => {
+    if (!this._picker) {
+      return
+    }
+    const node = ReactDOM.findDOMNode(this._picker)
```

Comment:
Can't think of a better solution than this

Reply:
@chrisnojima says this is OK for now, even though we're deprecating findDOMNode, since the long-term plan is to use our own emoji picker anyway.